js/utils/src/utils.ts

```diff
@@ -151,11 +151,6 @@ async function copy_to_clipboard(value: string): Promise<boolean> {
 	return copied;
 }
 
-// import { format } from "svelte-i18n";
```

Comment:
Is this intentional?

Reply:
it seemed like an unused comment so I removed it. am I missing something? cc @hannahblair just in case